AlfrescoSDK/AlfrescoSDK/Session/OnPremiseSession/AlfrescoRepositorySession.m

```diff
@@ -72,14 +65,35 @@ + (AlfrescoRequest *)connectWithUrl:(NSURL *)url
                          parameters:(NSDictionary *)parameters
                     completionBlock:(AlfrescoSessionCompletionBlock)completionBlock
 {
-    [AlfrescoErrors assertArgumentNotNil:url argumentName:@"url"];
-    [AlfrescoErrors assertArgumentNotNil:username argumentName:@"username"];
-    [AlfrescoErrors assertArgumentNotNil:password argumentName:@"password"];
-    [AlfrescoErrors assertArgumentNotNil:completionBlock argumentName:@"completionBlock"];
-    AlfrescoRepositorySession *sessionInstance = [[AlfrescoRepositorySession alloc] initWithUrl:url parameters:parameters];
-    if (sessionInstance) 
+    NSDictionary *startDict = [AlfrescoConnectionDiagnosticUtils createDictionaryForStartEventForEventName:kAlfrescoConfigurationDiagnosticReachabilityEvent];
+    [[NSNotificationCenter defaultCenter] postNotificationName:kAlfrescoConfigurationDiagnosticDidStartEventNotification object:nil userInfo:startDict];
```

Comment:
Yes, however, if the request was made on a background thread the notification would be send on the background thread. The user would have to manage the thread and ensure any UI code was sent to the main thread.

I just thought it would be better to send it to the main thread, and let the user do anything they want on a separate thread (if on the rare occasion they needed to do anything other then update UI). I believe most notifications sent by Apple are also sent on the main thread.